edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Docker/SystemResourcesMetrics.cs

```diff
@@ -144,6 +144,7 @@ async Task Update()
             // edgelet sets used cpu to -1 on error
             if (systemResources.UsedCpu > 0)
             {
+                Console.WriteLine($"host cpu {systemResources.UsedCpu}");
```

Comment:
Is this temporary debug output that can be removed? I ask because you used Console.WriteLine instead of the logger.